### PR TITLE
Feature/#997 pdf export styled md units

### LINKED ISSUE #997: PDF-Export: Styles for md-units
# User Story

### As a: 
User

### I want:
export pdfs including md-units with style

See #867 

## Acceptance criteria:
- [ ] MD in PDF-Exports are styled

### PR DESCRIPTION
**Pull request by Gargamil**
## Description:

Closes #997 

## Improvements
- styled pdf export for md-units

## Known Issues:
_NONE_

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `extractSingleMongoId` variant of the `ExtractMongoId` utility function(s). [#989](https://github.com/geli-lms/geli/issues/989)
 - Show message count for unit chat. [#933](https://github.com/geli-lms/geli/issues/993)
 - Styles for free text units. [#867](https://github.com/geli-lms/geli/issues/867)
+- Export pdf with styled md units [#997](https://github.com/geli-lms/geli/issues/997)
```
Comment (danielkesselberg) [inline CHANGELOG.md]: Please insert a dot at the end of the sentence.

**Comment by danielkesselberg** [comment]
image

I used the demo content from https://markdown-it.github.io/ for testing and selected `individual pdf - one pdf per unit`

**Comment by danielkesselberg** [comment]
> image
> 
> I used the demo content from https://markdown-it.github.io/ for testing and selected `individual pdf - one pdf per unit`

https://github.com/geli-lms/geli/pull/1014/commits/140bd7c0a587c1088395210c6c28db36489ff300 does the trick for me :sunglasses:

**Comment by danielkesselberg** [comment]
[1_What-is-Lorem-Ipsum_theme3.pdf](https://github.com/geli-lms/geli/files/2605340/1_What-is-Lorem-Ipsum_theme3.pdf)
[1_What-is-Lorem-Ipsum_default.pdf](https://github.com/geli-lms/geli/files/2605341/1_What-is-Lorem-Ipsum_default.pdf)



Looks good to me.

**Comment by Gargamil** [comment]
Ups. Sorry for that bug...  
Was blind for this because developed and tested only with single pdf -.-

**Pull request merged**